src/robot/parsing/model/statements.py

```diff
@@ -911,7 +911,7 @@ class Error(Statement):
     @property
     def errors(self):
         """Errors got from the underlying ``ERROR`` tokens."""
-        return tuple(t.error for t in self.get_tokens(Token.ERROR)) + self._errors
+        return tuple(t.error for t in self.get_tokens(Token.ERROR, Token.FATAL_ERROR)) + self._errors
```

Comment:
This line looks pretty long. CONTRIBUTING.rst states that the max line length with out Python code is 88 characters. Not a big problem can can be changed later.